src/Symfony/Component/Stopwatch/Tests/StopwatchTest.php

```diff
@@ -71,11 +69,12 @@ public function testStop()
     {
         $stopwatch = new Stopwatch();
         $stopwatch->start('foo', 'cat');
-        usleep(200000);
+        usleep(100000);
         $event = $stopwatch->stop('foo');
-
+        $duration = $event->getDuration();
+        usleep(100000);
         $this->assertInstanceof('Symfony\Component\Stopwatch\StopwatchEvent', $event);
-        $this->assertEquals(200, $event->getDuration(), null, self::DELTA);
+        $this->assertEquals($duration, $event->getDuration());
```

Comment:
@stof he is just saying that you are not testing the method anymore with your changes. And indeed, it looks wrong to me. You just call `getDuration()` twice, and the second time, you check it returns the same as the first call. That does not test if the returned value is actually correct.